css/style.css

```diff
@@ -266,12 +269,18 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: grid;
```

Comment:
To me, this was a nice example of a multi-column layout and I feel `grid` is apt here too.